src/plugins/lightbox/_base.scss

```diff
@@ -126,3 +126,17 @@ body {
 		@extend %global-display-block-important;
 	}
 }
+
+.mfp-wrap {
+	dialog{
```

Comment:
Is those css are the same as the CSS class `.mfp-container`? Can we just add the CSS class `mfp-container` on the dialog element.

Reply:
no, it is not the same. We can't just add class mfp-container